### DISCUSSION TITLE
Now News Channel

### DISCUSSION THREAD

**Discussion by darrencycheung** [iptv-org/community#339] [Show and tell]
### Channel ID

nowNews.hk

### Stream URL

https://pull-l3-cny.douyincdn.com/live/f49db49809a6d8e96047011ee9160bc8.m3u8

### Notes

Valid one

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://pull-l3-cny.douyincdn.com/live/f49db49809a6d8e96047011ee9160bc8.m3u8 (offline)
<img width="594" alt="image" src="https://user-images.githubusercontent.com/7253922/232168878-cc40993f-9d6e-4617-9573-8badcbc6d231.png">
